tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/fix_unmapped_survey_answers_test.py

```diff
@@ -0,0 +1,184 @@
+        
+        Question concept_id: 1332756 (How long has it been since you last used an electronic 
+        nicotine product?). value_source_concept_id : 1332724 (Months), which is non-standard but 
+        can be mapped to a standard concept_id through Maps to. The domain of the corresponding 
+        concept_id is 'Unit'. Set value_as_concept_id to the mapped concept_id (9580) 
+        
+        Question concept_id: 1586151 (Asian: Asian Specific). value_source_concept_id : 1585609 (
+        Asian Specific: Asian Specific Indian), which is non-standard but can be mapped to a 
+        standard concept_id through Maps to. However, the domain of the corresponding concept_id 
+        is 'Question', so we don't include this mapping and set value_as_concept_id to 0. 
+        
+        Question concept_id: 0 (No matching concept). value_source_concept_id : 1585536 (Yes), 
+        which is a deprecated concept_id and set value_as_concept_id to 0. 
+        
+        :return: 
+        """
+
+        concept_empl = self.jinja_env.from_string("""
+         INSERT INTO `{{fq_dataset_name}}.concept`
+            (concept_id,concept_name,domain_id,vocabulary_id,concept_class_id,standard_concept,concept_code,valid_start_date,valid_end_date,invalid_reason)
+            VALUES
+                (1585536,'Yes','Meas Value','PPI','Answer',null,'JacksonHindsAppointment_Yes','2017-07-26','2018-07-31','D'),
+                (1384615,'Respiratory Conditions: No Lung Condition','Observation','PPI','Answer','S','RespiratoryConditions_NoLungCondition', '2018-10-10', '2099-12-31',null),
+                (43530243,'Spoken To General Doctor: Yes','Observation','PPI','Answer',null,'SpokenToGeneralDoctor_Yes','2018-01-31','2099-12-31',null),
+                (1332776,'Moderately unhappy','Observation','PPI','Answer',null,'cope_a_31','2020-05-07','2099-12-31',null),
+                (1585609,'Asian Specific: Asian Specific Indian','Observation','PPI','Answer',null,'AsianSpecific_AsianSpecificIndian','2017-05-22','2099-12-31',null),
+                (1384515,'Kidney Conditions: Kidney Stones','Observation','PPI','Answer',null,'KidneyConditions_KidneyStones','2018-10-09','2099-12-31',null),
+                (4179221,'History of calculus of kidney','Observation','SNOMED','Context-dependent','S','429025008','2008-01-31','2099-12-31',null),
+                (4088548,'Seen by general practitioner','Observation','SNOMED','Clinical Finding','S','185278000','1970-01-01','2099-12-31',null),
+                (1586151,'Asian: Asian Specific','Observation','PPI','Question','S','Asian_AsianSpecific','2017-05-17','2099-12-31',null),
+                (1333079,'Moderately unhappy','Observation','PPI','Answer','S','mhwb_a_40','2020-05-07','2099-12-31',null),
+                (9580,'month','Unit','UCUM','Unit','S','mo','1970-01-01','2099-12-31',null),
+                (1332724,'Months','Observation','PPI','Answer',null,'cope_a_198','2020-05-07','2099-12-31',null)
+        """)
```

Comment:
To piggy back on Mark's comment for max line length of 100, I also configured my editor so it complains about this too.

Reply:
Yeah my editor is complaining but in this case I don't think I'd want the line length <= 100. I guess I'll try to make my editor not complain on test scripts. :man_shrugging: